images/archive/importer.go

```diff
@@ -330,7 +330,7 @@ func compressBlob(ctx context.Context, cs content.Store, r io.Reader, ref string
 	}
 
 	defer func() {
-		w.Close()
+		err = w.Close()
```

Comment:
The `err` is used to return.  It is used to rollback. 
We can't override it ....

Reply:
Understood, thanks